### ISSUE TITLE
[BUG] Fatal Exception: java.lang.NullPointerException

### ISSUE THREAD

**Issue by themadmrj** [jhomlala/betterplayer#530]
**History check**
I did.

**Describe the bug**

This is a fatal crash report (app crash) from firebase crashlytics from the app in production. Therefore, I cannot give you more than what the report says. I'm guessing an additionnal check should be enough to avoid the crash.

```
Fatal Exception: java.lang.NullPointerException
Attempt to invoke virtual method 'void com.google.android.exoplayer2.upstream.n0.l.b()' on a null object reference


com.jhomlala.better_player.CacheWorker.onStopped (CacheWorker.java:2)
androidx.work.ListenableWorker.stop (ListenableWorker.java:3)
androidx.work.impl.WorkerWrapper.interrupt (WorkerWrapper.java:28)
androidx.work.impl.Processor.interrupt (Processor.java:4)
androidx.work.impl.Processor.stopWork (Processor.java:34)
androidx.work.impl.utils.StopWorkRunnable.run (StopWorkRunnable.java:75)
androidx.work.impl.utils.SerialExecutor$Task.run (SerialExecutor.java:2)
java.util.concurrent.ThreadPoolExecutor.processTask (ThreadPoolExecutor.java:1187)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1152)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
java.lang.Thread.run (Thread.java:784)
```

**Expected behavior**
A silent error is fine, but a fatal crash is not.

**Better Player version**
- Version: 0.0.68 (I'm guessing there was no change in the last update regarding this bug).

**Additional context**
Here are the specs:
Device
Brand:HUAWEI
Model:HUAWEI P smart 2020
Orientation: Portrait
RAM free: 1.06 GB
Disk free: 77.04 GB
Operating System
Version:9
Orientation: Portrait
Rooted:No
Date:4 Jun 2021, 14:12:56

**Comment by jhomlala** [comment]
Seems like some random crash. CacheWorker in onStopped has this code:

```java
 mCacheWriter.cancel();
 super.onStopped();
```

Basically we can only apply try catch here. I'll do that in next version.

**Comment by themadmrj** [comment]
Seems legit, thanks :)

**Comment by jhomlala** [comment]
Fixed in 0.0.70.
